wallaby.js

```diff
@@ -16,7 +16,6 @@ function setup({projectCacheDir, testFramework}) {
 module.exports = wallaby => ({
   files: [
     'jest.config.js',
-    'scripts/setup-test-framework.js',
```

Comment:
Don't we need the file anymore?

Reply:
we already deleted this file previously and forgot about this entry